pkg/apis/pipeline/v1beta1/pipeline_conversion.go

```diff
@@ -259,6 +259,14 @@ func (m *Matrix) convertTo(ctx context.Context, sink *v1.Matrix) {
 		param.convertTo(ctx, &new)
 		sink.Params = append(sink.Params, new)
 	}
+	for _, include := range m.Include {
```

Comment:
```suggestion
	for i, include := range m.Include {
```
we might use the index here for line 267